Automattic/jetpack
# Don't re-send emails when marking responses as not-spam

Fixes #30305.

## Proposed changes:

This patch will prevent emails from being send when items are being restored from spam on the new forms dashboard.

### Other information:


## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






- Go to `/wp-admin/admin.php?page=jetpack-forms`.
- Make sure there are some responses with valid emails (that you have access to) and contact details and mark some of them as spam.
- Go to the spam tab.
- Select those responses and remove them from spam.
- You should not be getting any emails.
